### ISSUE TITLE
Keyboard accessibility

### ISSUE THREAD

**Issue by barrystoner** [jamietre/ImageMapster#150]
Hello,

Great plugin. Thank you for sharing as this could be extremely useful for me.

Out of the box it seems to break the keyboard accessibility of my page and I can neither tab to the areas or tab past them. So I effectively get stuck on the screen. I would love to able to use this plugin, but if it's not accessible using a keyboard only then it's a problem for me.

Hopefully it's something simple that I need to include in my code.

Thank you in advance.

Barry
